backend/src/nodes/nodes/rest/img2img.py

```diff
@@ -0,0 +1,99 @@
+from __future__ import annotations
+
+import numpy as np
+
+from . import category as RESTCategory
+from ...impl.rest import (
+    decode_base64_image,
+    SamplerName,
+    STABLE_DIFFUSION_IMG2IMG_URL,
+    post_async,
+    encode_base64_image,
+)
+from ...node_base import AsyncNodeBase
+from ...node_factory import NodeFactory
+from ...properties.inputs import (
+    TextInput,
+    NumberInput,
+    EnumInput,
+    ImageInput,
+)
+from ...properties.outputs import LargeImageOutput
+
+
+@NodeFactory.register("chainner:rest:sd_img2img")
+class Img2Img(AsyncNodeBase):
+    def __init__(self):
+        super().__init__()
+        self.description = "Modify an image using an external Stable Diffusion service"
+        self.inputs = [
+            ImageInput(),
+            TextInput("Prompt", default="an astronaut riding a horse"),
+            TextInput("Negative Prompt", default=""),
+            NumberInput(
+                "Denoising Strength",
+                minimum=0,
+                default=0.75,
+                maximum=1,
+                controls_step=0.1,
+                precision=4,
+            ),
```

Comment:
Generally, sliders are nicer for users because they visualize not only the current value but also the relation of the current value to the minimum and maximum.

![image](https://user-images.githubusercontent.com/20878432/214319177-81516431-fd58-4863-bca6-7a8530eb56b7.png)
```suggestion
            SliderInput(
                "Denoising Strength",
                minimum=0,
                default=0.75,
                maximum=1,
                slider_step=0.01,
                controls_step=0.1,
                precision=4,
            ),
```